experimental/auth-logic/wrappers/rekor_wrapper_test.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package wrappers
+
+import (
+	"testing"
+)
+
+const testRekorLogPath = "experimental/auth-logic/test_data/rekor_entry.json"
+
+func TestRekoLogWrapper(t *testing.T) {
+	logEntryAnon, err := getLogEntryAnonFromFile(testRekorLogPath)
+	if err != nil {
+		t.Fatalf("couldn't parse rekor log entry from path: %s. %v", testRekorLogPath, err)
+	}
+
+	entryImpl, err := getEntryImplFromAnon(*logEntryAnon)
+	if err != nil {
+		t.Fatalf("couldn't get entryImpl from body of logEntryAnon logEntryAnon: %v, rekordLogFilePath: %s. err: %v", *logEntryAnon, testRekorLogPath, err)
+	}
+
+	rekordEntry, err := getRekordEntryFromEntryImpl(*entryImpl)
+	if err != nil {
+		t.Fatalf("couldn't get rekordEntry from entryImpl. entryImpl: %v, rekordLogFilePath: %s. err: %v", *entryImpl, testRekorLogPath, err)
+	}
+
+	_, err = verifyRekordLogSignature(rekordEntry)
```

Comment:
I suppose this will be repeated, more or less as-is, in the implementation of the wrapper. I'd suggest wrapping all this in a higher-level function (named for instance `verifyRekorLogEntry`) and writing the test for that.

Reply:
The test scenario in #77 seems to have some redundancy, but I only had a quick look at the code. Ideally, we want a test for `EmitStatement`, and perhaps additional, detailed tests for some of the functions called in `EmitStatement`. 

Also, the code in `EmitStatement` is repeating the code in the test here. This is the part that I think can be refactored into a single function, tested here, and called in `EmitStatement` in #77.

Reply:
Since this PR is already merged, what do you want me to do?

In #77 the wrapper also makes the authorization logic statements next to the go code so that it's easier to follow what the policy statement means. If we pack this into a single `verifyRekorLogEntry` we can't do that anymore.

Reply:
> If we pack this into a single verifyRekorLogEntry we can't do that anymore.

I think it should still be possible, but let me have a closer look at #77 (possibly tomorrow). I think this change can be done as part of #77, or perhaps a separate PR after that.